api/RTCIceTransport.json

```diff
@@ -5,43 +5,43 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceTransport",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": false
           },
           "chrome": {
-            "version_added": null
+            "version_added": false
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": false
           },
           "edge": {
-            "version_added": null
+            "version_added": "15"
           },
           "edge_mobile": {
-            "version_added": null
+            "version_added": false
           },
           "firefox": {
-            "version_added": null
+            "version_added": false
```

Comment:
The title of the PR implies that only Safari iOS versions are changed. However, you have also set other browsers' `version_added` to `false`. This should be incorporated into the PR title.

Reply:
Ah, I just saw that the changes are also in #2461, so there probably in here by accident. You might want to remove the commit from the `update-safari-ios-versions` branch.